src/Compilers/Core/Portable/StrongName/StrongNameFileSystem.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class StrongNameFileSystem
+    {
+        internal virtual byte[] ReadAllBytes(string fullPath)
+        {
+            Debug.Assert(PathUtilities.IsAbsolute(fullPath));
+            return File.ReadAllBytes(fullPath);
+        }
+
+        /// <summary>
+        /// Resolves assembly strong name key file path.
+        /// Internal for testing.
```

Comment:
`Internal for testing` comment is no longer needed since there are other `internal` callers.

Reply:
Done.